lib/travis/addons/serializer/pusher/build.rb

```diff
@@ -102,7 +102,9 @@ def build_data
                 author_name: commit.author_name,
                 author_email: commit.author_email,
                 committer_name: commit.committer_name,
-                committer_email: commit.committer_email
+                committer_email: commit.committer_email,
```

Comment:
I could actually delete these as they’ll no longer be used in `travis-web` after [this PR is merged](https://github.com/travis-ci/travis-web/pull/1740)… thoughts?

Reply:
in theory others (e.g. our CLI, or external parties) could use it ... i'd probably just leave it